consensus/consensus_leader_msg.go

```diff
@@ -11,6 +11,52 @@ import (
 	"github.com/harmony-one/harmony/internal/utils"
 )
 
+// LeaderNetworkMessage ..
+type LeaderNetworkMessage struct {
+	Phase                      quorum.Phase
+	Bytes                      []byte
+	OptionalAggregateSignature *bls.Sign
+}
+
+// TODO Finish refactoring other three message constructions folded into this function.
```

Comment:
is a ticket to keep track of this TODO? please add your name here.

Reply:
No specific ticket, just the task of implementing double signing slashing.